### PR TITLE
Bump version number for GA

### PR DESCRIPTION
**Pull request by pherl**
@haberman @jskeet also for review.

### CODE REVIEW

**Comment by jskeet** [comment]
C# changes LGTM.

File: ruby/google-protobuf.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = "google-protobuf"
-  s.version     = "3.0.0.alpha.7.0.0"
+  s.version     = "3.0.0.0.0.0"
```
Comment (xfxyjwf) [inline ruby/google-protobuf.gemspec]: Do we need that many 0s here?

File: ruby/pom.xml [hunk 2]
```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-beta-3</version>
+            <version>3.0.0</version>
```
Comment (xfxyjwf) [inline ruby/pom.xml]: This will need to stay as 3.0.0-beta-4 until we release Java 3.0.0 to maven. Changing it to 3.0.0 now will break the jruby build.

**Comment by xfxyjwf** [comment]
Travis is experience some issues: https://www.traviscistatus.com/ . I wasn't even able to cancel some other queued tests.

We probably have to rely on Jenkins tests for now.

File: ruby/pom.xml (hunk 2 above)
Comment (pherl) [inline ruby/pom.xml]: Done.

File: ruby/google-protobuf.gemspec (hunk 1 above)
Comment (pherl) [inline ruby/google-protobuf.gemspec]: Removed.

**Comment by xfxyjwf** [comment]
LGTM

**Pull request merged**